### PR TITLE
Updated Changelog and README

### LINKED ISSUE #97: Update Changelog and README
Before release 0.0.3 update changelog:

- [x] Add update on documentation to generate correct plots.
- [x] Improved performance with `WarnOnce`.

### PR DESCRIPTION
**Pull request by ojeda-e**
## Description
Fixes #97 

Changes in this PR include:
- [x] Updated changelog
- [x] Updated README

## Status
- [x] Ready to go

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,24 +44,26 @@ python setup.py install
...
 
-or via pip:
+or via `pip`:
 
 ```
 pip install --upgrade MDAnalysisTests
 ```
 
+> ⚠️ In comparison to the previous version, `membrane-curvature==0.0.3` shows a significant improvement in performance, particularly notable for membrane-protein systems. Installing the last available version is highly encouraged. 
+
```
Comment (ojeda-e) [inline README.md:60]: Now that I read this section again I wonder if I should clarify that the package is available for Python 3.10 as well. I never said it explicitly, neither here nor in docs. What do you think @hmacdope? Maybe adding it at the beginning of the installation section?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,12 @@
 Membranecurvature CHANGELOG
 =============================
 
+0.0.3 (18-10-2022)
+-------------------
+* Fixed warning messages that affected performance (PR #95).
+* Fixed plots in documentation pages and updated plots in tutorialsss (PR #89).
```
Comment (hmacdope) [inline CHANGELOG.rst:7]:
```suggestion
* Fixed plots in documentation pages and updated plots in tutorials (PR #89).
```

File: README.md (hunk 1 above)
Comment (hmacdope) [reply README.md:60]: Its in the changelog in this PR, so i think its ok.

**Review by hmacdope — CHANGES_REQUESTED** [review]

**Comment by IAlibay** [comment]
Just FYI, if you're about to do a release - 3.11 is just around the corner, I'd advise waiting for it.

**Comment by ojeda-e** [comment]
> Just FYI, if you're about to do a release - 3.11 is just around the corner, I'd advise waiting for it.

Thanks for pointing this out @IAlibay 
I will wait for the release and meanwhile I can create an issue to ` '3.11.0-alpha` to the CI, if that''s ok.

**Comment by IAlibay** [comment]
> > Just FYI, if you're about to do a release - 3.11 is just around the corner, I'd advise waiting for it.
> 
> Thanks for pointing this out @IAlibay 
> I will wait for the release and meanwhile I can create an issue to ` '3.11.0-alpha` to the CI, if that''s ok.

3.11.0-rc.2 is probably what I'd aim for here. You'll need to use a pure python action rather than mamba (see mda_xdrlib or gh-ci-cron in core).

File: CHANGELOG.rst (hunk 2 above)
Comment (ojeda-e) [reply CHANGELOG.rst]: Ugh, thanks for catching this.

**Comment by ojeda-e** [comment]
Thanks for the patience @hmacdope. I think this PR is ready for your review.

**Review by hmacdope — APPROVED** [review]
LGTM @ojeda-e

**Pull request merged**